saas/api/serializers.py

```diff
@@ -757,6 +756,17 @@ class PlanCreateSerializer(PlanDetailSerializer):
     class Meta(PlanDetailSerializer.Meta):
         fields = PlanDetailSerializer.Meta.fields
 
+    def validate(self, data):
+        period_type = Plan.INTERVAL_CHOICES[data['period_type'] - 1][1]
+        period_amount = data.get('period_amount')
+
+        min_amount = getattr(settings, f'BROKER_MINIMUM_PLAN_AMOUNT_{period_type}', 0)
```

Comment:
I would most likely have used a dictionary lookup instead of a text substitution here. Maybe just a preference.